core/src/components/item/test/disabled/index.html

```diff
@@ -82,13 +78,7 @@
           </ion-list-header>
 
           <ion-item>
-            <ion-checkbox aria-label="Checkbox" disabled slot="start"></ion-checkbox>
-            <ion-input aria-label="Input" placeholder="Disabled Checkbox w/ Input"></ion-input>
-          </ion-item>
-
-          <ion-item>
-            <ion-checkbox aria-label="Checkbox" disabled slot="end"></ion-checkbox>
-            <ion-input aria-label="Input" placeholder="Disabled Checkbox w/ Input"></ion-input>
+            <ion-input aria-label="Input" placeholder="Input"></ion-input>
```

Comment:
The items changed here were all originally checking multiple different inputs in one item, with one being disabled and the other not. This is lost now that the checkboxes have been removed. If we don't think these checks are still useful, we should remove the items entirely.

Reply:
Brandy is taking care of this in https://github.com/ionic-team/ionic-framework/pull/29017 which is why I didn't remove the inputs.

Reply:
Brandy's PR only covers the `ion-input`s; other items changed here use radios, a range, a toggle, a button, and a searchbar, which this comment still applies to. Even accounting for future PRs that remove the other legacy components, there's still the button and searchbar.

Maybe there should be a separate PR that removes the section entirely so we don't have as many screenshot diffs for what is intended to be a temporary state? Both here and in the other PRs removing legacy component usage from this test.

Reply:
I also noticed that Brandy's PR just removes the relevant items completely. That might be a better alternative; it feels less confusing than leaving the items in a temporary state and changing the labels.

Reply:
Good idea - updated in https://github.com/ionic-team/ionic-framework/pull/29020/commits/19af7b5b5bcb2bb49bcc7186cd122119c3c895a3